unlock-app/src/components/interface/checkout/FormStyles.tsx

```diff
@@ -96,6 +97,33 @@ export const LoadingButton = ({ children, ...props }: LoadingButtonProps) => (
   </Button>
 )
 
+interface TransactionPendingButtonProps {
+  network: number
+  transaction: any
```

Comment:
why is transaction an any? 

I like the refactor though. It could more generalized.

Reply:
Fixed type!

I think it could be a great addition to the `ui` package eventually?